addons/l10n_id/models/account_move.py

```diff
@@ -0,0 +1,47 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+from odoo.tools.misc import formatLang
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def _compute_tax_totals(self):
+        """ OVERRIDE
+
+        For invoices based on ID company as of January 2025, there is a separate tax base computation for nun-luxury goods.
+        Tax base is supposed to be 11/12 of original while other tax bases should remain the same.
+
+        To avoid affecting calculation of total amount, set tax_group_amount to 0 but the formatted_tax_group_amount to
+        the DPP value
+        """
+        super()._compute_tax_totals()
+        dpp_label = "DPP Other Value (11 / 12)"
+        for move in self.filtered(lambda m: m.is_sale_document()):
+            non_luxury_tax_group_id = self.env.ref('l10n_id.l10n_id_tax_group_non_luxury_goods', raise_if_not_found=False).id
+            if not (move.invoice_date and move.invoice_date < fields.Date.to_date('2025-01-01')):
+                dpp_other_value = 0
+                # for non-luxury taxes, accumulate total adjusted base amount
+                for subtotal in move.tax_totals['groups_by_subtotal']:
+                    subtotal_group = move.tax_totals['groups_by_subtotal'][subtotal]
+                    for group in subtotal_group:
+                        # For components with the non-luxury tax group, adjust the base amount for `formatted_tax_group_base_amount`
+                        # to avoid changing the total calculation
+                        if group['tax_group_id'] == non_luxury_tax_group_id:
+                            dpp = group['tax_group_base_amount'] * (11 / 12)
+                            dpp_other_value += dpp
+                            group['formatted_tax_group_base_amount'] = formatLang(self.env, dpp, currency_obj=move.currency_id)
+                if dpp_other_value > 0:
+                    label = move.tax_totals['subtotals_order'][0]
+                    group = move.tax_totals['groups_by_subtotal']
+                    dpp_group = {
+                        "tax_group_name": dpp_label,
+                        "tax_group_amount": 0,
```

Comment:
Would this not get rid of the need of the js file? Then `formatData` of `TaxTotalsCompent` should just do the right thing already?
```suggestion
                        "tax_group_amount": dpp_other_value,
```